ISM43362Interface.cpp

```diff
@@ -337,7 +321,6 @@ void ISM43362Interface::socket_check_read()
                 /* Check if there is something to read for this socket. But if it */
                 /* has already been read : don't read again */
                 if ((socket->connected) && (socket->read_data_size == 0) && _cbs[socket->id].callback) {
-                    _ism.setTimeout(1);
```

Comment:
How will we avoid being blocked in the receive process. We only want to check if there is data, don't want to wait for it. 
Should we call to setRecvTimeout   ?